controllers/getArticleByID.controller.js

```diff
@@ -0,0 +1,17 @@
+const articleByID = require('../models/getArticleByID.model')
+
+function getArticleByID(req, res, next) {
+    const { article_id } = req.params
+    
+    articleByID(article_id).then((article) => {
+        if (!article){
+            res.status(404).send({message: 'Not Found'})
+        }
```

Comment:
Instead of having conditional logic here I'd consider having this as Promise.reject in the model if no articles are found. This will keep any extra complexity free from the controller and allow your error handlers to pick it up.